datastore/google/cloud/datastore/_http.py

```diff
@@ -14,46 +14,21 @@
 
 """Connections to Google Cloud Datastore API servers."""
 
-import contextlib
 import os
 
 from google.rpc import status_pb2
 
-from google.cloud._helpers import make_insecure_stub
-from google.cloud._helpers import make_secure_stub
 from google.cloud import _http as connection_module
 from google.cloud.environment_vars import DISABLE_GRPC
 from google.cloud.environment_vars import GCD_HOST
 from google.cloud import exceptions
 from google.cloud.grpc.datastore.v1 import datastore_pb2 as _datastore_pb2
 try:
-    from grpc import StatusCode
-    from google.cloud.grpc.datastore.v1 import datastore_pb2_grpc
+    from google.cloud.datastore._gax import _DatastoreAPIOverGRPC
+    _HAVE_GRPC = True
```

Comment:
So, the ultimate idea is that:

  - Developer imports `from x.y.z.v1.client import Client`
  - The `client` module tries to import `_gax`, wrapped in a try-except, and multiplexes based on whether GRPC is available?